lib/CleantalkSP/SpbctWP/Scanner/Surface.php

```diff
@@ -474,4 +482,40 @@ public static function dirIsEmpty($dir)
 
         return true;
     }
+
+    public function filterFileExtensionUsingRegexp($extension, array $extensions)
+    {
+        foreach ($extensions as $extensionItem) {
+            $fsymbol = strpos($extensionItem, '[');
+            $lsymbol = strpos($extensionItem, ']');
+
+            $regexpExpression = substr($extensionItem, $fsymbol, $lsymbol);
+            if ($regexpExpression) {
```

Comment:
Please add the condition `Helper::isRegexp()` checking too.

Reply:
Done